### PR TITLE
support for Node.js v14

### PR DESCRIPTION
**Pull request by bensooraj**
I was inspired by this [tweet](https://twitter.com/openwhisk/status/1259907986471952385) to make a contribution to this project!

Please let me know if something's not right.

### CODE REVIEW

**Comment by rabbah** [comment]
Wow @bensooraj thanks for the contribution!

**Comment by dgrove-oss** [comment]
Very cool!  @bensooraj could you file a Contributor License Agreement with the ASF as described in: https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md

**Comment by rabbah** [comment]
I have amended the PR with a commit to add doc/test changes to cover the new runtime. We can accept this PR without a CLA since the delta in the first commit is actually quite small (I diff'ed against node v12). You're still welcome and encourage to submit a CLA @bensooraj if you might contribute more in the future.

File: README.md [hunk 1]
```diff
@@ -56,13 +57,20 @@ wsk action update myAction myAction.js --kind nodejs:10
 wsk action update myAction myAction.js --kind nodejs:12
 ```
 
+### Node.js v14
+
+```
+wsk action update myAction myAction.js --kind nodejs:14
```
Comment (rabbah) [inline README.md:63]: This won't work until we actually add the runtime to OpenWhisk but I added it anyway in anticipation of that happening soon.

**Comment by bensooraj** [comment]
Apologies, looks like I missed a lot!

Sure, I will go ahead submit a CLA.

Thanks!

**Pull request merged**